samples/compose-samples/src/main/java/com/squareup/sample/compose/hellocomposeworkflow/ComposeWorkflow.kt

```diff
@@ -40,21 +40,21 @@ public abstract class ComposeWorkflow<in PropsT, out OutputT : Any> :
    * workflow's parent.
    * @param viewEnvironment The [ViewEnvironment] passed down through the `ViewBinding` pipeline.
    */
-  @Composable public abstract fun RenderingContent(
+  @Composable abstract fun RenderingContent(
```

Comment:
Why no public?

Reply:
Oops! Good catch, thanks.

Reply:
Oh, because this is a sample, no explicit API mode -- `public` is redundant.